### PR TITLE
Clean up lints

### PR DESCRIPTION
**Pull request by joebingham-wk**
Working on CPLAT-12194, I noticed there were just a few small lints that could be cleaned up just to have analysis clean.

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://dev.workiva.net/docs/teams/information-security/aviary#configuring-aviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

**Review by aaronlademann-wf — APPROVED** [review]
+1

File: test/over_react_redux/connect_test.dart [hunk 1]
```diff
@@ -70,17 +70,17 @@ main() {
 
     test('throws when multiple `mapStateToProps` type args are provided', () {
       expect(() => connect<CounterState, CounterProps>(
-          mapStateToProps: (state){},
-          makeMapStateToProps: (state, ownProps){}
+          mapStateToProps: (state) => {},
+          makeMapStateToProps: (state, ownProps) => (_) => {}
```
Comment (kealjones-wk) [inline test/over_react_redux/connect_test.dart:74]: What exactly does this do? like what was the lint? this seems pointless. if this were JS sure it binds the `this` context but here it just means like `return`... was the lint like `missing-return-of-non-void-return-value-cause-im-dart`?

**Review by kealjones-wk — APPROVED** [review]
QA +1

File: test/over_react_redux/connect_test.dart (hunk 1 above)
Comment (joebingham-wk) [reply test/over_react_redux/connect_test.dart:74]: Haha yep, exactly @kealjones-wk, just a missing return statement that should return a function

**Comment by joebingham-wk** [comment]
@Workiva/release-management-p

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**